auslib/test/admin/views/test_releases.py

```diff
@@ -779,11 +730,10 @@ def testPostRevisionRollback(self):
 
         query = table.history.t.count()
         count, = query.execute().first()
-        self.assertEqual(count, 2 * 2 + 1)
+        self.assertEqual(count, 3)
 
         row, = table.select(where=[table.name == 'd'])
-        self.assertEqual(row['version'], '222.0')
-        self.assertEqual(row['data_version'], 6)
+        self.assertEqual(row['data_version'], 4)
```

Comment:
Lets add a check that the release blob has been reverted properly. Ideally we can tell the difference between test setup --> True --> False --> <reverted value>, so might need to swap to something numerical over boolean.